main.js

```diff
@@ -110,9 +111,12 @@ define(function (require, exports, module) {
 	addIcon('py', 'ion-social-python', '#f8c63d');
 	addAlias('pyw', 'py');
 
+	// Qt Quick
+	addIcon('qml', 'ion-code', '#42ed0e');
+
 	// Shell and friends
-	addIcon('sh', 'ion-document-text', '#008d00');
```

Comment:
Can you switch back to the old color? Or, if you have a better color, use that? My design is that extensions in the same 'family' share the same icon, but the icons have different color. For example, all images use ion-image but have different colors.